src/js/components/Search.jsx

```diff
@@ -112,13 +112,31 @@ class Search extends Component {
 
   onChangeCategory (ev, index, category) {
     const {dispatch} = this.props
+    // const xarray = []
```

Comment:
i would get rid of all these comments. if they are meant to be a part of another branch, than just cut/paste them over to the other branch.